build.sbt

```diff
@@ -2,11 +2,12 @@ organization in ThisBuild := "com.lightbend"
 
 name := """akka-sample-cluster-kubernetes-scala"""
 
-version := "0.1"
-
 scalaVersion := "2.13.0"
 lazy val akkaHttpVersion = "10.1.11"
-lazy val akkaVersion    = "2.6.5"
+lazy val akkaVersion = "2.6.5"
+
+version in ThisBuild ~= (_.replace('+', '-'))
+dynver in ThisBuild ~= (_.replace('+', '-'))
```

Comment:
I had problem with `docker:publishLocal` when updating (after changes). It hang on deleting some stuff. Had to restart Docker Desktop. I think this could have helped, but not quite sure what was going on.

Reply:
FOr dynver can set dynverSeparator in ThisBuild := "-"